### PR TITLE
Some chages of modloader.py

### PR DESCRIPTION
**Pull request by mrsteyk**
Just some changes of modloader.py to make it somewhat less aggressive trying to load modules.
And important newline, you guidline faggot.

### CODE REVIEW

**Comment by mrsteyk** [comment]
NOW IT HAS MOAR BS IN IT!!!

File: modloader.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def load_modules(updater):
     api = TelegramAPI(updater)
 
     for module_name in listdir('./modules'):
-        if module_name.endswith('.py'):
+        if not module_name.startswith('lib-') and module_name.endswith('.py'):
```
Comment (evgfilim1) [inline modloader.py:12]: What's the reason to add this check?

**Comment by evgfilim1** [comment]
Btw, `requirements.txt`, you've mistyped

**Comment by mrsteyk** [comment]
@evgfilim1 Then it means octo also mistyped

**Comment by evgfilim1** [comment]
@mrsteyk Yes

File: modloader.py (hunk 1 above)
Comment (mrsteyk) [reply modloader.py:12]: to not be shitty and have libs if they are needed. Or just use it to disable module 😄.

**Comment by mrsteyk** [comment]
bled. i fucked up.

**Comment by mrsteyk** [comment]
если хочешь сам напишешь

**Comment by evgfilim1** [comment]
@mrsteyk okay

File: modloader.py (hunk 1 above)
Comment (evgfilim1) [reply modloader.py:12]: It'll be loaded anyway

**Pull request closed without merging**